datafusion/sqllogictest/test_files/map.slt

```diff
@@ -45,7 +45,7 @@ describe data;
 ----
 ints Map("entries": Struct("key": Utf8, "value": Int64), unsorted) NO
 strings Map("entries": Struct("key": Utf8, "value": Utf8), unsorted) NO
-timestamp Utf8View NO
+timestamp Utf8 NO
```

Comment:
There are other tests too because of this setting: `datafusion.execution.parquet.schema_force_view_types false`